presto-hive/src/test/java/com/facebook/presto/hive/HiveTestUtils.java

```diff
@@ -267,4 +269,28 @@ public static Slice longDecimal(String value)
     {
         return encodeScaledValue(new BigDecimal(value));
     }
+
+    public static Optional<String> getProperty(String name)
+    {
+        String systemPropertyValue = System.getProperty(name);
+        String environmentVariableValue = System.getenv(name);
+        if (systemPropertyValue == null) {
+            if (environmentVariableValue == null) {
+                return Optional.empty();
```

Comment:
if you're returning optional values, and the result is `null`, why not use `Optional.ofNullable`?

Reply:
Changed to your rewritings anyways.